docker.md

```diff
@@ -4,6 +4,23 @@ To build the docker image, run the following command:
 
 `docker build -t "arc:latest" .`
 
-To run the docker image, run the following command:
+To run the docker image for testing, run the following command:
```

Comment:
Why not change it to:
To run the docker image for self-hosting, run the following command:

Reply:
because the inline docker CLI command given is not data-persistant. For self-hosting, people should use Docker Compose.